.yarn/build-state.yml

```diff
@@ -0,0 +1,18 @@
+# Warning: This file is automatically generated. Removing it is fine, but will
```

Comment:
This file shouldn't be commited

Reply:
Thanks. It was added before I update the '.gitignore' file.
Fixed